### PR TITLE
updated line length in code contrib section

### LINKED ISSUE #347: update line length in Contributor guide
The Contributor Guide in https://userguide.mdanalysis.org/stable/contributing_code.html#code-formatting-in-python states 79 char line limit but we actually use 88 chars (see `.flake8` https://github.com/MDAnalysis/mdanalysis/blob/f4005db383f3f4c446433fb32156236f29f251ba/.flake8#L2 )

The contributor guide should be updated to say 88 chars per line.

### PR DESCRIPTION
**Pull request by orbeckst**
- fix #347
- add note on black

----
:books: Documentation preview :books:: https://mdanalysisuserguide--348.org.readthedocs.build/en/348/

### CODE REVIEW

File: doc/source/contributing_code.rst [hunk 1]
```diff
@@ -515,6 +516,20 @@ Or if you're ok with the suggested formatting changes, just apply the suggested
     darker -r upstream/develop testsuite/MDAnalysisTests -L flake8
 
 
+.. note::
+
+   MDAnalysis does *not* currently use the popular `black`_ code formatter across the whole project, therefore please consider the following rules:
+
+   * Do **not run ``black`` on an existing file** that you are editing. This makes the diff for the PR very difficult to read. Instead use ``darker`` as explained above to only reformat your changes.
```
Comment (lilyminium) [inline doc/source/contributing_code.rst]: Some interaction with bolding makes the backticks not render properly, unfortunately.

```suggestion
   * **Do not run** ``black`` on an existing file that you are editing. This makes the diff for the PR very difficult to read. Instead use ``darker`` as explained above to only reformat your changes.
```

**Review by lilyminium — CHANGES_REQUESTED** [review]
Thanks for the change, @orbeckst! Just one minor change.

**Comment by orbeckst** [comment]
Sorry @lilyminium this one totally slipped off my radar. Thank you for the suggested fix.

**Review by lilyminium — APPROVED** [review]
LGTM, thank you for picking this up again @orbeckst!

**Pull request merged**